### ISSUE TITLE
Observed data import: Use smallest LLOQ value for each data set

### ISSUE THREAD

**Issue by TWendl** [Open-Systems-Pharmacology/OSPSuite.Core#1524]

**Comment by msevestre** [comment]
FYI @PavelBal  and @Yuri05

**Comment by msevestre** [comment]
I am not convinced but I am probably thinking backwards

Say you have the following values
0.3, <0.2, < 0.5

With the current implementation, we would assume lloq 0.5 and all values would be below lloq

If we take 0.2 instead, then 0.3 becomes a valid value? And < 0.5 gets the value 0.1? (lloq/2) which is lower than the 0.25 it would receive normally

Just want to be sure 😅

**Comment by PavelBal** [comment]
I disagree. This is a (hopefully) very rare use case, where one data set would have different LLOQs. Actually, it should not happen at all. We should assume that one data set is measured in the same lab with the same assay - and LLOQ is a property of the assay.

Now say for whatever reason the data set has different LLOQs - 0.3 and 0.5. But if the assay cannot quantify anything below 0.5, it definitely also cannot quantify 0.4 and 0.3! Setting the LLOQ to 0.3 would save 0.4 as is but it does not make sense, because the assay already cannot quantify 0.5!
